src/components/CardComponentList.tsx

```diff
@@ -1,46 +1,45 @@
-import { ResponseData } from '../interfaces/songInterface';
-import Image from 'next/image';
-import { useMediaQuery } from 'react-responsive';
+import { ResponseData } from "../interfaces/songInterface";
+import Image from "next/image";
+import { useMediaQuery } from "react-responsive";
 
 const CardList: React.FC<ResponseData> = (data) => {
-    const isMobile = useMediaQuery({ query: '(max-width: 767px)' });
-    return (
-        <div className="xl:h-[50vh] lg:h-[60vh] overflow-hidden">
-            {/* remember to have a default height/width if using media breakpoints */}
-            <div className="overflow-y-scroll lg:h-[50vh] md:h-[60vh] h-[80vh]">
-                <ul className="divide-y divide-gray-200">
-                    {data.items.map((item, index) => (
-                        <li key={index} className="py-4">
-                            <div className="flex flex-row items-center">
+  const isMobile = useMediaQuery({ query: "(max-width: 767px)" });
+  return (
+    <div className="xl:h-[50vh] lg:h-[60vh] overflow-hidden">
+      {/* remember to have a default height/width if using media breakpoints */}
+      <div className="overflow-y-scroll lg:h-[50vh] md:h-[60vh] h-[80vh]">
+        <ul className="divide-y divide-gray-200">
+          {data.items.map((item, index) => (
+            <li key={index} className="py-4">
```

Comment:
Try using something other than index here as the key.

https://react.dev/learn/rendering-lists#rules-of-keys

Reply:
using item.id instead